codec-memcache/src/main/java/io/netty/handler/codec/memcache/MemcacheContent.java

```diff
@@ -27,22 +27,4 @@
  * in the {@link ChannelPipeline}.
  */
 public interface MemcacheContent extends MemcacheObject, ByteBufHolder {
-
-    @Override
-    MemcacheContent copy();
-
-    @Override
-    MemcacheContent duplicate();
-
-    @Override
-    MemcacheContent retain();
-
-    @Override
-    MemcacheContent retain(int increment);
-
-    @Override
-    MemcacheContent touch();
-
-    @Override
-    MemcacheContent touch(Object hint);
```

Comment:
same comment as above. This is needed as otherwise it will not return the correct type and so not allow method chaining correctly.